pages/api/docs/getRepoList.ts

```diff
@@ -4,7 +4,7 @@ import { getUserRepositories } from "../../../utils/providerAPI/getUserRepositor
 import { authOptions } from '../auth/[...nextauth]';
 import { RepoIdentifier } from '../../../types/repository';
 
-const getUserRepositoriesHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+const getRepoListHandler = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
Why change this name without any change in the functionality? Doesn't the previous name explain what the function is doing better?